pkg/config/config.go

```diff
@@ -25,6 +25,7 @@ type Constants struct {
 	HmacKeyLength                  			int
 	CORSAccessControlAllowOrigin   			string
 	DisableCurrentDateCheckFeatureFlag	bool
+	EnableEntirePeriodBundle						bool
```

Comment:
Super minor, and maybe this is just the github editor, but it doesn't line up with the others. Do we have a linter / formatter?

Reply:
Fixed